google/cloud/aiplatform/experimental/vertex_model/utils/source_utils.py

```diff
@@ -75,14 +96,43 @@ def _make_source(
         the user has the option to specify a method to call from __main__.
     """
 
+    module = inspect.getmodule(obj.__class__)
```

Comment:
Double checking that this is supported in Jupyter and Colab environments where the class is defined in the Notebook?

Reply:
Let me double check in the Jupyter notebook before pushing this code.